package/kernel/linux/modules/netdevices.mk

```diff
@@ -716,13 +716,30 @@ define KernelPackage/ixgbevf
   AUTOLOAD:=$(call AutoLoad,35,ixgbevf)
 endef
 
+
 define KernelPackage/ixgbevf/description
  Kernel modules for Intel(R) 82599 Virtual Function Ethernet adapters.
 endef
 
 $(eval $(call KernelPackage,ixgbevf))
 
 
+define KernelPackage/amd-xgbe
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:=AMD Ethernet support
+  DEPENDS:=@PCI_SUPPORT +kmod-lib-crc32c +kmod-ptp +kmod-libphy
```

Comment:
I'd have this depend on x86, or even x86_64 if this is only for x86_64 systems.

Like this:

`DEPENDS:=@TARGET_x86 @PCI_SUPPORT +kmod-lib-crc32c +kmod-ptp +kmod-libphy`

or `@TARGET_x86_64` if this only applies to 64 bit systems.